subiquity/controllers/zdev.py

```diff
@@ -633,7 +637,15 @@ def typeclass(self):
 
 class ZdevController(SubiquityController):
 
+    autoinstall_key = "s390x-devices"
+    autoinstall_default = ()
+    autoinstall_schema = {
+        'type': 'array',
+        'items': {'type': 'string'},
```

Comment:
Somewhere we did document schema, and wanted `enable`, `disable`, `ignore`. For chzdev enable, czdev disable, and cio_ignore. As on large LPARs cio_ignore=all makes the boot so much faster.

Disable is needed to disable autoactivated devices, or like call enable all followed by disable one or two things one doesn't want. The latter is very rare.

Reply:
I don't remember anything about a more detailed schema I'm afraid, so I'm happy to believe anything you come up with is a design I've already agreed to :)